modules/masterbar.php

```diff
@@ -20,3 +20,18 @@
 
 new Masterbar();
 new Admin_Color_Schemes();
+
+/**
+ * Whether to load the admin menu functionality.
+ *
+ * @use add_filter( 'jetpack_load_admin_menu_class', '__return_true' );
+ * @module masterbar
+ *
+ * @since 9.2.0
```

Comment:
This is going to be 9.3.0 now

```suggestion
 * @since 9.3.0
```

Reply:
e4dfd0e6ccb471a0aca44985c282f408f02adf2e